src/main/features/claim/views/fees.njk

```diff
@@ -55,9 +55,9 @@
               </thead>
               {% for row in rows %}
                 <tr>
-                  <td>{{ row.from | pennies2pounds | numeral }} to {{ row.to | pennies2pounds | numeral }}</td>
-                  <td>{{ row.fees[1] | pennies2pounds | numeral }}</td>
-                  <td>{{ row.fees[2] | pennies2pounds | numeral }}</td>
+                  <td>{{ row.minRange }} to {{ row.maxRange }}</td>
+                  <td>{{ row.fees[1] | default('Not available') }}</td>
```

Comment:
why is this default needed?

Reply:
For an out-n-out edge case when fee register messes up hearing fee. Currently they're streamlining ranges but provided us with upper limit in the interim but that may change.

Reply:
The 'Not available' should be translated into Welsh.

Reply:
@tarik-johnston1 confirmed it is translatable to Welsh and fee table without pennies is fine too.